index.html

```diff
@@ -0,0 +1,147 @@
+        </div>
+    </nav>
+    <header class="navbar fixed-top-nav" id="top-header">
+        <div class="container">
+            <i class="fa-solid fa-bars" id="menu-icon"></i>
+            <div class="desktop-nav">
+                <img class="logo-nav" src="./images/aspire-logo.png" width="110px" alt="Aspire logo">
+                <ul class="nav-list float-right">
+                    <li><a class="menu-list-item" href="about.html">About</a></li><br>
+                    <li><a class="menu-list-item" href="#about-myself">Program</a></li><br>
+                    <li><a class="menu-list-item" href="#my-form">Join</a></li><br>
+                    <li><a class="menu-list-item" href="#my-form">Sponsor</a></li><br>
+                    <li><a class="menu-list-item" href="#my-form">News</a></li><br>
+                    <li><a href="#" class="mt-3  p-2" id="btn-nav">Company</a></li><br>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <nav class="mobile-nav">
+        <span class="close-button">x</span>
+        <ul class="menu-list-mobile">
+            <li><a class="menu-list-item" href="about.html">About</a></li><br>
+            <li><a class="menu-list-item" href="#main-program">Program</a></li><br>
+            <li><a class="menu-list-item" href="#join">Join</a></li><br>
+            <li><a class="menu-list-item" href="#sponsor">Sponsor</a></li><br>
+            <li><a class="menu-list-item" href="#news">News</a></li><br>
+            <li><a href="#" class="mt-3  p-2" id="btn-nav">BP Launch</a></li><br>
+        </ul>
+    </nav>
+
+    <main class="main-section">
+        <section class="top-bg">
+            <section class="hero container">
+                <br>
+                <h4 class="greetings">"Explore Your Dream Career with Aspire"</h4>
+                <h2 class="hero-title">Get connected <br /> to the experts Globally.</h2>
+                <br>
+            </section>
+
+            <section class="message">
+
+                <div class="container">
+                    <p class="message-p p-3 border border-white">
+                        At Career Guiders, we help young people explore different career options with the help of
+                        industry experts. Our mission is to reduce unemployment in Africa by funding talents and
+                        connecting young people with experts in their field of interest. Join us today and start your
+                        journey to success!
+                    </p>
+                    <h3 class="message-time">24/7 Active Services</h3>
+                    <p>@Rwanda Kigali converntion center</p>
+                </div>
+            </section>
+            <br><br><br>
+        </section>
+        <section class="main-program" id="main-program">
+            <br>
+            <h4 class="text-center section-title-white">Main Program</h4>
+            <hr class="title-line">
+            <div class="program-holder container">
+
```

Comment:
- [ ] According to the project requirement the pages should look almost identical to the original design, pages should use the exact same colors, for text, backgrounds, borders, etc. I would suggest you please use the `Fonts: Cocogoose, Lato` or similar fonts from the original design.  The font uses for the heading, sub heading is different and also the paragraph is different so I would suggest you please use the same color and font. See the image below for reference.
- [ ] Also Please adjust the spacing to match the original design this will help you follow the exact design in future.
### Your design


![his](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/81727988/54610fa7-5ff3-4a8c-9bf4-e2e311e77271)


### Original design
![original](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/81727988/d49870b9-92ac-4342-84d5-64e17a659323)